docker/Dockerfile

```diff
@@ -118,6 +118,7 @@ RUN --mount=type=cache,target=/root/.cache \
 # purposes. This script installs these things. (We use `-u` for immediate
 # unbuffered output).
 RUN --mount=type=bind,source=scripts/install_duckdb_extras.py,target=/root/install_duckdb_extras.py \
+    --mount=type=bind,source=scripts/ddb-view,target=/root/ddb-view \
```

Comment:
Why not use `COPY`?

Reply:
The `install_duckdb_extras.py` script does the copying, this is just making the file available to that script. However on looking at this again, it's a bit nasty that the Dockerfile has to know so much about the dependencies of the script. So I've updated it to mount the whole `scripts/` directory instead. That does mean we might invalidate this layer slightly more often than needs be (whenever _any_ file in `scripts/` changes) but I don't think that matters much and the decrease in complexity is worth it.